amy/consents/models.py

```diff
@@ -85,6 +99,19 @@ class TermOption(CreatedUpdatedArchivedMixin, models.Model):
     def __str__(self):
         return f"{self.content} ({self.option_type})"
 
+    def archive(self) -> None:
+        """
+        Archive self and archive all Consent objects that have the
+        term option as their answer.
+        If the Term this term option is attached to is still active,
```

Comment:
~~Something wrong with this sentence.~~
Edit: scratch that, something wrong was with me not the sentence 😅